package.json

```diff
@@ -6,14 +6,16 @@
     "ios": "expo start --ios",
     "lint": "./node_modules/.bin/eslint __tests__ src --ext .js,.jsx,.ts,.tsx",
     "start": "expo start",
-    "test": "jest",
+    "test": "node_modules/.bin/jest",
+    "test:debug": "node --inspect-brk node_modules/jest/bin/jest.js --runInBand",
+    "test:coverage": "node_modules/.bin/jest --coverage",
     "build:android": "expo build:android",
     "build:ios": "expo build:ios"
   },
   "jest": {
-    "preset": "react-native",
+    "preset": "jest-expo",
```

Comment:
was this change found somewhere in the expo docs?

Reply:
https://jestjs.io/docs/25.x/tutorial-react-native no docu has "react-native" . 
This change comes from montas package.json

Reply:
> no docu has "react-native"

how do you mean that? the one you have linked has the "react-native" preset.

Reply:
the expo documentation has the `jest-expo` preset: https://docs.expo.io/guides/testing-with-jest/

the documentation also says to install `jest-expo`, which we have, but we also have just `jest`.

please see if the `jest` package is actually needed, and remove it, if it is unnecessary

Reply:
215113225b7d1604db7f8eeff6b7cf951ee91319